lib/cache/checkCacheOptions.ts

```diff
@@ -0,0 +1,33 @@
+import { CacheOptions } from '.';
+
+export function checkCacheOptions(options: CacheOptions): void {
```

Comment:
`checkCacheOptions`? Maybe just `checkOptions`, as we're already in cache scope.
https://github.com/labs42io/clean-code-typescript#dont-add-unneeded-context